src/openrct2/world/Map.cpp

```diff
@@ -2227,3 +2232,176 @@ MapRange ClampRangeWithinMap(const MapRange& range)
     MapRange validRange = MapRange{ aX, aY, bX, bY };
     return validRange;
 }
+
+void ShiftMap(const TileCoordsXY& amount)
+{
+    if (amount.x == 0 || amount.y == 0)
+        return;
```

Comment:
Shouldn't this be using `&&`? Now if either x or y is zero, it won't shift anything.